app/scripts.babel/friends.js

```diff
@@ -204,4 +209,12 @@ init({ friends: true, cachedRanks: true }, ({ friends, cachedRanks }) => {
       replaceRanklist();
     }
   });
+}
+
+observer.forEach(SELECTOR_TO_HIDE, elem => elem.style.display = 'none');
```

Comment:
If we hide only `.ranklist`, the footer will jump down during page loading. I think this doesn't look good.